benchmark_data_loading/common.py

```diff
@@ -62,9 +62,14 @@ def bench(f, inp, num_exp=3, warmup=1, unit="μ", num_images_per_call=DATASET_SI
 
 def iterate_one_epoch(obj):
     if isinstance(obj, (data.datapipes.datapipe.IterDataPipe, FFCVLoader, data.DataLoader, DataLoader2)):
+        i = 0
         for _ in obj:
-            pass
+            i += 1
+        if i == 0:
+            raise RuntimeError("Dataset contains no samples!")
```

Comment:
We probably don't need this for now?

Reply:
I'm ok without since I now know the reason if I'm getting strange benchmarks. Still, it does not hurt to leave it in, does it?

Reply:
I'd rather keep the code as simple as possible (although benchmark code is always a bit of a mess I would agree)